docs/configuration/config-file-reference.md

```diff
@@ -741,11 +741,6 @@ results_cache:
     # The CLI flags prefix for this block config is: frontend
     [fifocache: <fifo_cache_config>]
 
-  # Most recent allowed cacheable result, to prevent caching very recent results
```

Comment:
Should we outright remove it from docs or should we mark it DEPRECATED, but keep it in docs?

Reply:
I would keep it simple and keep in the doc only the current version of the flags (the ones expected to be used).